source/components/organisms/SummaryList/SummaryList.tsx

```diff
@@ -101,14 +102,18 @@ const SummaryList: React.FC<Props> = ({
       const oldAnswer: Record<string, string | number | boolean>[] = answers[item.id];
       oldAnswer[index][item.inputId] = value;
       onChange(oldAnswer, item.id);
+    } else if (['editableListText', 'editableListNumber', 'editableListDate'].includes(item.type) && item.inputId) {
+      const oldAnswer: Record<string, string | number | boolean> = answers[item.id];
+      oldAnswer[item.inputId] = value;
+      onChange(oldAnswer, item.id);
     } else {
       onChange(value, item.id);
     }
   };
 
   const onItemBlur = (item: SummaryListItem, index?: number) => (value: string | number | boolean) => {
     if (
-      ['arrayNumber', 'arrayText', 'arrayDate'].includes(item.type) &&
+      ['arrayNumber', 'arrayText', 'arrayDate', 'editableListText', 'editableListNumber', 'editableListDate'].includes(item.type) &&
```

Comment:
array.... brus

Reply:
array might not be a good prefix, but I need something, because text, number and date are already used and references other types of inputs. array here means that the answer comes in an array, from a repeater.

Reply:
hmm. ok.